lib/gc/Target/LLVM/CMakeLists.txt

```
@@ -0,0 +1,17 @@
+add_mlir_dialect_library(MLIRGENTarget
+  GEN/Target.cpp
+
+  OBJECT
+
+  ADDITIONAL_HEADER_DIRS
+  ${MLIR_MAIN_INCLUDE_DIR}/mlir/Dialect/LLVMIR
+  ${PROJECT_SOURCE_DIR}/include/gc/Dialect/LLVMIR
+
+  LINK_LIBS PUBLIC
+  MLIRIR
+  MLIRExecutionEngineUtils
+  MLIRSupport
+  MLIRGPUDialect
+  MLIRTargetLLVM
+  MLIRGENToLLVMIRTranslation
+  )
```

```suggestion
  LLVMSPIRVCodeGen
  )
```
Otherwise, linker fails with undefined symbol: LLVMInitializeSPIRVTarget

---

This is weird, I don't see that failure locally somehow. Will take a closer look.

---

I've rebased to the main branch and the LLVM build failed, because SPIRV is broken in that version, as I mentioned above. Then I've update the LLVM version, successfully built it, but the GC build failed with the linkage error.